apps/arena/lib/arena/game/player.ex

```diff
@@ -282,4 +320,35 @@ defmodule Arena.Game.Player do
     end
     |> Enum.uniq()
   end
+
+  defp apply_effect({:stamina_faster, stamina_faster}, player) do
```

Comment:
This mechanic is not working correctly, it has a bug when you apply this mechanic multiple times, first the stamina is being reduced and restores correctly but if you use the same item twice you end up with less stamina cooldown permanently, i'll explain myself in a video and a diagram:

The stamina bug is due to the usage of message to restore the stamina, the second message to restore the stamina to a lowest value as the original will be received before the one to restore the stamina to a lowest value

![bug stamina](https://github.com/lambdaclass/mirra_backend/assets/106101218/0f4d4e23-71ca-4620-b023-348fb01e5c58)

Here i'm using a lot of clocks and the bug is real:
the left terminal is printing the current player stamina value


https://github.com/lambdaclass/mirra_backend/assets/106101218/b8ceb2ca-b5b5-499a-becd-3fce4f81ae35